core/modules/qmeta/QMetaMysql.h

```diff
@@ -185,8 +224,8 @@ class QMetaMysql : public QMeta {
      *  @param queryId:   Query ID, non-negative number.
      *  @return: List of query IDs.
      */
-    virtual std::vector<int> getQueriesForTable(std::string const& dbName,
-                                                std::string const& tableName);
+    virtual std::vector<QueryId> getQueriesForTable(std::string const& dbName,
+                                                    std::string const& tableName);
 
 protected:
 
```

Comment:
I did not know that doxygen can do that, thanks for idea. Having said that I mostly care about myself as a developer, I prefer to have documentation near the method declaration when I read source instead of chasing bunch of other files for interface that defined that virtual method first (good IDE could probably help with that but I don't have it). 

Should we ask about particular guideline for doxygen and inherited methods, I don't see anything in our current guidelines.